spec/models/user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  before do
+    @user = build(:user)
+  end
```

Comment:
変数をテスト内で参照する場合はletの方がいいかと思います。

```suggestion
  let(:user) { build(:user)  }
```

Reply:
letとbeforeについて調べてみましたが、インスタンス変数を用いないほうがエラーログ参照などで利点があることが分かりました。letだと参照範囲が、user_spec.rb内なので、今回はletで問題ないということですね。
こちら、letに変更します。

Reply:
すいません、説明不足でしたね。
調べていただいた内容の通りです。実装としては問題ないけど、letの方が利点が大きいです。
あと参照しない時には実行(キャッシュ)されないので、若干速度的に早くなる(かも)。